bc_obps/registration/tests/integration/test_operation_registration.py

```diff
@@ -226,6 +236,12 @@ def test_operation_registration_workflow(self, operation_type, purpose):
             #### Opted-In Operation Detail Form ####
             self._set_opted_in_operation_detail()
 
+        print("-->Documents")
+        print(self.operation.documents.all())
+
```

Comment:
left behind

Reply:
Pulled it out when I reset to remove the breaking Vite bump.